src/mapper/pkg/resolvers/schema.helpers.resolvers.go

```diff
@@ -151,55 +149,7 @@ func (r *Resolver) addSocketScanServiceIntent(ctx context.Context, srcSvcIdentit
 }
 
 func (r *Resolver) resolveOtterizeIdentityForService(ctx context.Context, svc *corev1.Service, lastSeen time.Time) (model.OtterizeServiceIdentity, bool, error) {
```

Comment:
consider eliminating this function (which now has 0 extra logic) and just call `r.kubeFinder.ResolverOtterizeIdentity...` instead.